go/porcelain/deploy.go

```diff
@@ -979,6 +979,7 @@ func createHeader(archive *zip.Writer, i os.FileInfo, runtime string) (io.Writer
 		return archive.CreateHeader(&zip.FileHeader{
 			CreatorVersion: 3 << 8,     // indicates Unix
 			ExternalAttrs:  0777 << 16, // -rwxrwxrwx file permissions
+			Comment:        i.Name(),
```

Comment:
Tiny one, but could we add a comment here so that future us are not 🤯 trying to figure out why?

Reply:
yes! done in [22d8ab9](https://github.com/netlify/open-api/pull/509/commits/22d8ab916d0baf122c9a71f35a8349bba759f269)